sorts/src/main/java/za/co/oldtimerza/sorts/InsertionSort.java

```diff
@@ -0,0 +1,31 @@
+package za.co.oldtimerza.sorts;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class InsertionSort {
+    public static List<Integer> sort(List<Integer> sortableList){
```

Comment:
avoid static in this instace.

We want to make instances of these so that we can follow the strategy pattern in the future.

In fact a good practice to do now that we have a few sorting algorithms is to extract a "Sort" interface.

Perhaps something like:


```suggestion
    public interface Sort {
         List<Integer> sort(List<Integer> sortableList);
    }
```